Decimus/Lib/libquicr/FullTrackName.swift

```diff
@@ -47,7 +47,8 @@ class FullTrackName: QFullTrackName, Hashable {
     /// - Returns: ASCII string of namespace.
     /// - Throws: ``FullTrackNameError/parseError`` if ``namespace`` is not ecodable as ASCII.
     func getNamespace() throws -> String {
-        guard let namespace = String(data: self.nameSpace, encoding: .ascii) else {
+        guard let element = self.nameSpace.first,
+              let namespace = String(data: element, encoding: .ascii) else {
             throw FullTrackNameError.parseError
         }
         return namespace
```

Comment:
Looks like this only returns the string of the first entry, should we concatenate all the entries with '/' for the full namespace?